Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Build:
+#   $ docker build -t yottadb:latest .
+# Use with data persistence:
+#   $ docker run --rm -e gtm_chset=utf-8 -v `pwd`/ydb-data:/data -ti yottadb:latest
```

Comment:
It would be nice to mention that the directory ybd-data would persist after the docker run command finishes. And one can clean this up if needed.